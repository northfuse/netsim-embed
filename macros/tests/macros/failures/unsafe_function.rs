#[netsim_embed_macros::machine]
unsafe fn foo() {}

fn main() {}
