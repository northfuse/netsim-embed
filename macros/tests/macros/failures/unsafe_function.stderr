error: custom attribute panicked
 --> tests/macros/failures/unsafe_function.rs:1:1
  |
1 | #[netsim_embed_macros::machine]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: netsim_embed::machine cannot be unsafe
